src/Lykke.Service.SwiftWithdrawal/Workflow/Projections/SwiftStateChangedProjection.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Lykke.Common.Chaos;
+using Lykke.Service.PersonalData.Contract;
+using Lykke.Service.SwiftWithdrawal.Contracts;
+using Lykke.Service.SwiftWithdrawal.Core.Domain.Cashout;
+
+namespace Lykke.Service.SwiftWithdrawal.Workflow.Projections
+{
+    public class SwiftStateChangedProjection
```

Comment:
Consider to rename to SwiftRequestLogProjection and use SwiftCashoutCreatedEvent